src/components/loader/loader-bar.spec.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import TestRenderer from "react-test-renderer";
+import { mount } from "enzyme";
+import StyledLoaderBar, { InnerBar } from "./loader-bar.style";
+import { assertStyleMatch } from "../../__spec_helper__/test-utils";
+import baseTheme from "../../style/themes/base";
+import LoaderBar from "./loader-bar.component";
+
+function render(props) {
+  return TestRenderer.create(<StyledLoaderBar {...props} />);
```

Comment:
We're trying to slowly remove our `TestRenderer` tests - can you use enzyme instead please